pkg/math/oracle/median.go

```diff
@@ -121,56 +111,45 @@ func (m *MedianAggregator) AggregateData() {
 // The prices utilized are the prices most recently seen by the providers. Each price is within a
 // MaxPriceAge window so is safe to use.
 func (m *MedianAggregator) CalculateConvertedPrices(
-	target mmtypes.Ticker,
-	paths mmtypes.Paths,
+	market mmtypes.Market,
 ) []*big.Int {
-	m.logger.Debug("calculating converted prices", zap.String("ticker", target.String()))
-	if len(paths.Paths) == 0 {
+	m.logger.Debug("calculating converted prices", zap.String("ticker", market.Ticker.String()))
+	if len(market.ProviderConfigs) == 0 {
 		m.logger.Error(
-			"no conversion paths",
-			zap.String("ticker", target.String()),
+			"no provider configs",
+			zap.String("ticker", market.Ticker.String()),
 		)
 
 		return nil
 	}
 
-	convertedPrices := make([]*big.Int, 0, len(paths.Paths))
-	for _, path := range paths.Paths {
-		if len(path.Operations) == 0 {
-			m.logger.Error(
-				"no operations",
-				zap.String("ticker", target.String()),
-			)
-
-			continue
-		}
-
+	convertedPrices := make([]*big.Int, 0, len(market.ProviderConfigs))
+	for _, config := range market.ProviderConfigs {
```

Comment:
This pre-alloc on the size won't affect any price calculations right?

Reply:
yes, this is just the "capacity" not the length

slice initailization is `make(type, len, capacity)`